src/configuration/configuration_controller.py

```diff
@@ -0,0 +1,88 @@
+from plumbum import local, FG, BG, TF, RETCODE, ProcessExecutionError
+import subprocess
+
+
+class ConfiguartionController:
+    def __init__(self):
+        self._multichain_util = local['multichain-util']
+        self._multichain_d = 'multichaind'
```

Comment:
Nit: Please be consistent with how you declare strings. If you’re going to use double-quotes for strings, keep using that.

Reply:
![](https://media.giphy.com/media/4baoNZ5Qo8dX2/giphy.gif)

All right, I’ll take your word on it.